roles/deploy/tasks/main.yml

```diff
@@ -86,7 +86,7 @@
     force=yes
     owner={{ unicorn_user }}
     mode=0775
-  when: newrelic_license_key != "none"
+  when: newrelic_license_key is defined
```

Comment:
is still someone using new relic? I believe most of us are using Skylight already.